testpmd/hooks/pre-run.yml

```diff
@@ -37,7 +37,6 @@
     packet_generator_networks:
       - name: intel-numa0-net2
         count: 2
-    example_cnf_deploy_script_version: master
```

Comment:
Hmmm, this one is needed, even when the default is the same (master) this is the place we override it when doing tests (example: https://github.com/dci-labs/example-cnf-config/pull/36/files) to validate changes in the "upstream" repo: https://github.com/rh-nfv-int/testpmd-operator

I'm adding this one back

Reply:
And now I'm removing it because I see this one was moved to the pipeline, so we are good :+1: